tests/phpunit/includes/bootstrap.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 $_root_dir = dirname( dirname( dirname( __DIR__ ) ) );
-$_tests_dir = $_root_dir . '/tmp/wordpress-tests-lib';
+$_tests_dir = ! empty( getenv( 'WP_TESTS_DIR' ) ) ? getenv( 'WP_TESTS_DIR' ) : $_root_dir . '/tmp/wordpress-tests-lib';
```

Comment:
This comes from [here](https://github.com/polylang/polylang/blob/346af790ad646b1b18b1946803fdc2dda2329834/.github/actions/phpunit/action.yml#L47)
Also, when using `wp-env` in the future (e.g. coming with e2e tests), this will allow us to run PHPUnit from within our local docker containers.